src/include/btree.h

```diff
@@ -242,6 +242,7 @@ struct __wt_btree {
      * code.
      */
     WT_REF *evict_ref;                         /* Eviction thread's location */
+    uint linear_walk_restarts;                  /* next/prev walk restarts */
```

Comment:
May be change to uint32_t to be consistent with the following variables.

Reply:
Done